draft-srld-teas-5g-slicing.md

```diff
@@ -263,11 +265,6 @@ The term "Transport Network" is used for disambiguation with 5G network (e.g., I
       for logical separation, Quality of Service (QoS), or Traffic
       Engineering (TE).
 
-      A 5G network slicing architecture
-      should integrate TN Slicing for an optimal control of SLAs, however, it is
-      possible to implement 5G Network Slicing without TN
-      Slicing, as explained in section {{{#sec-mapping}}}.
-
```

Comment:
is it removed (I am looking at the diff) ?

Reply:
Yes, it is removed.